config/dev.json

```diff
@@ -4,6 +4,7 @@
   "https": {
     "enabled": false
   },
+  "memoryLimit": 30,
```

Comment:
What's the unit of `memoryLimit`? Percent?

Reply:
yes percent